src/main/java/radiounit/AlarmStatusLevel.java

```diff
@@ -1,7 +1,4 @@
-/**
- * 
- */
-package common;
+package radiounit;
```

Comment:
This should stay in common - it'll be used elsewhere other than just in the radio unit

Reply:
Sounds good, I'll move it back. I wasn't positive it was going to be used by anything else.